package/Dockerfile.agent

```diff
@@ -21,10 +43,10 @@ LABEL io.cattle.agent true
 ENV AGENT_IMAGE rancher/rancher-agent:${VERSION}
 ENV CATTLE_RANCHER_WEBHOOK_MIN_VERSION=''
 # For now, this value needs to be manually synced with the one in the main Dockerfile. This pins downstream webhook's version.
-ENV CATTLE_RANCHER_WEBHOOK_VERSION=2.0.12+up0.3.12-rc1
+ENV CATTLE_RANCHER_WEBHOOK_VERSION=2.0.11+up0.3.11
```

Comment:
```suggestion
ENV CATTLE_RANCHER_WEBHOOK_VERSION=2.0.12+up0.3.12-rc1
```

Reply:
This is the current non-RC image that is most up-to-date at the release-v2.7 branch for webhook.
The RC might have come from a dev branch and is no longer valid since they are deleted after some time.

Reply:
@tashima42